src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,69 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+public class StringClassTest {
+
+//  요구사항 1
+    @Test
+    @DisplayName("\"1,2\"을 ,로 split 했을 때 1과 2로 잘 분리되는지 확인하는 학습 테스트를 구현한다.")
+    void splitTest(){
+        String value = "1,2";
+        String[] actual = value.split(",");
+
+        String expectedComponent1 = "1";
+        assertThat(actual).contains(expectedComponent1);
+
+        String expectedComponent2 = "2";
+        assertThat(actual).contains(expectedComponent2);
```

Comment:
https://bcp0109.tistory.com/317

위글을 참고해서

```suggestion
        assertThat(actual).contains("1", "2");
```
이렇게 쓰는것도 좋을거 같기도 하네요~